docs/ref/contrib/postgres/search.txt

```diff
@@ -177,8 +175,18 @@ vector you wish to use. For example:
 This index will then be used by subsequent queries. In many cases this will be
 sufficient.
 
-If this approach becomes too slow, you can add a ``tsvector`` column to your
-table and use triggers to populate it. This approach is described in the
-`PostgreSQL documentation`_.
+``SearchVectorField``
+---------------------
+
+.. class:: SearchVectorField
+
+If this approach becomes too slow, you can add a ``SearchVectorField`` to your
+model and keep it populated. An approach to doing this with triggers is
```

Comment:
to your model. You'll need to keep it populated with triggers, for example, as described...